app/static/js/events/wizard/vue-app.js

```diff
@@ -350,10 +358,21 @@ function shouldDisableMove($this) {
         $this.event.start_time_time.trim() === '' ||
         $this.event.start_time_date.trim() === '' ||
         $this.event.end_time_time.trim() ==='' ||
-        $this.event.end_time_date.trim() === ''
+        $this.event.end_time_date.trim() === '' ||
+        checkTickets($this)
     );
 }
 
+function checkTickets($this) {
+    for( var ticket of $this.event.tickets){
```

Comment:
Don't use ES6 syntax here. Not all browsers support it. Use lodash's foreach to loop through.